introduction.md

```diff
@@ -8,4 +8,6 @@ As someone who always provides high quality of service, I am always trying to be
 The foundation of my managerial believes lies in my passion for project management and time management. 
 In every company I work on I bring fresh ideas and creative way of thinking, also help everybody to get to the peak of their potential.
 
-And BTW, CodeSchool rocks!!! My favorite path is Ruby
\ No newline at end of file
+And BTW, CodeSchool rocks!!! My favorite path is Ruby
+
+*Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?